server/openslides/users/serializers.py

```diff
@@ -119,6 +134,9 @@ def create(self, validated_data):
         inform_changed_data(user)
         return user
 
+    def get_vote_delegations_id(self, user):
```

Comment:
Maybe be consistent with the server naming and change it to `get_vote_delegated_from_id`.

Reply:
That was my first idea, too, but it bothered me that the plural is lost there and from the name you just assume its a single id.

Reply:
This is a pattern all in OS3. It would be clear if instead of using `_id`, use `users_id`.

Reply:
So `vote_delegated_from_users_id`?

Reply:
ok